index.html

```diff
@@ -31,7 +31,7 @@
 	<aside id="sticky-generator">
 	<a name="jump"></a> 
 			<h1>Sticky Generator</h1>
-			<div class="sticky-template">
+			<div class="sticky" id="sticky-template">
```

Comment:
K, I added [this](https://github.com/msweinberg/fewd-final-project/pull/1/files#diff-42f69825340f0b5fa628d80900a46daaR28)